.github/workflows/sub-deploy-integration-tests-gcp.yml

```diff
@@ -535,58 +369,6 @@ jobs:
           sudo journalctl -b \
```

Comment:
2 lines above this line we use `$DISK_NAME`, but it hasn't been written to `$GITHUB_ENV` by any previous step.

Reply:
Yeah, this comes from the main branch. I'll create another PR to improve this.